index.ts

```diff
@@ -365,8 +366,10 @@ export interface JUserService {
   getLocale(): string
   getPreference(name: string): string | null
   hasPreference(name: string): boolean
+  getDistanceUnit(projectId: number): JDistanceUnit | null
   removePreference(name: string): string | null
   setPreference(name: string, value: string | undefined): void
+  setDistanceUnit(projectId: number, distanceUnit: JDistanceUnit): void
```

Comment:
I'm sorry I'm rigid but please could you place getDistanceUnit not between 2 preference methods ! And better if the setDistanceUnit is right after the getDistanceUnit !

Reply:
put getDistanceUnit right above :)